packages/react/src/components/Modal/next/Modal.stories.js

```diff
@@ -0,0 +1,347 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import { action } from '@storybook/addon-actions';
+import {
+  boolean,
+  object,
+  optionsKnob as options,
+  select,
+  text,
+  withKnobs,
+} from '@storybook/addon-knobs';
```

Comment:
Knobs won't work in the new storybook, they've been remove in favor of [controls](https://storybook.js.org/docs/react/essentials/controls). We can remove the knobs code for now. We'll need to add controls at some point though.